app/router/index.js

```diff
@@ -28,4 +29,6 @@ router.post('/schema/diff', asyncWrap(schema.diff));
 router.delete('/schema/delete/:schemaId', asyncWrap(schema.delete));
 router.post('/schema/validate', asyncWrap(schema.validate));
 
+router.delete('/service/delete/:name', asyncWrap(service.delete));
```

Comment:
wouldn't it be more logical to name it `'/schema/delete/:name'`? I'm reading it as "schema delete service" vs. current "service delete service"

Reply:
hmm.. I would suggest to have a breaking change and to have different query params that would be deleting based on different variations

```
DELETE /schema?id=
DELETE /schema?serviceName=
```